ClientApp/src/components/pages/ProjectEditPage/ProjectEditPage.tsx

```diff
@@ -102,18 +103,23 @@ export const ProjectEditPage: React.FC = (): JSX.Element => {
     setFieldValue,
   } = useFormik<ProjectDetails>({
     initialValues: {
-      client: undefined,
-      endDate: undefined,
-      evaluationDeadline: undefined,
-      evaluations: [{ evaluationId: 0, projectRole: ProjectRole.PM }],
+      client: { clientId: -1, name: "" },
+      creator: fakeNoNameEmployeeSnippet,
+      endDate: "",
+      evaluationDeadline: "",
+      evaluationStatus: -1,
+      evaluations: [],
+      lastModifiedBy: fakeNoNameEmployeeSnippet,
       links: [
-        { linkId: undefined, name: "Jira Project", url: undefined },
-        { linkId: undefined, name: "Confluence Space", url: undefined },
-        { linkId: undefined, name: "Project Retro", url: undefined },
+        { linkId: -1, name: "Jira Project", url: "" },
+        { linkId: -1, name: "Confluence Space", url: "" },
+        { linkId: -1, name: "Project Retro", url: "" },
```

Comment:
```suggestion
        { linkId: 0, name: "Jira Project", url: "" },
        { linkId: 0, name: "Confluence Space", url: "" },
        { linkId: 0, name: "Project Retro", url: "" },
```
To fix 500 errors. If the entity doesn't exist yet, its id should be 0.